src/components/misc/DisplaySettings.tsx

```
@@ -0,0 +1,200 @@
+              {["Author", "MOTD", "Tags", "Players Online", "Actions"].map(
+                (property) => (
+                  <button
+                    key={property}
+                    onClick={() => toggleProperty(property)}
+                    className={cn(
+                      "px-3 py-1.5 text-sm rounded-md transition-colors",
+                      selectedProperties.includes(property)
+                        ? "bg-secondary text-secondary-foreground border"
+                        : "hover:bg-muted/80"
+                    )}
+                  >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)